wsmaster/codenvy-hosted-api-machine/src/test/java/com/codenvy/api/machine/server/spi/tck/RecipePermissionsDaoTest.java

```diff
@@ -224,6 +224,13 @@ public void shouldThrowExceptionWhenPermissionsExistsActionArgumentIsNull() thro
         dao.exists("user", "instance", null);
     }
 
+    @Test
+    public void shouldStoreRecipePermission() throws Exception {
+        final RecipePermissionsImpl recipePermissions = new RecipePermissionsImpl("*", "recipe1", Arrays.asList("read", "use", "run"));
+        dao.store(recipePermissions);
```

Comment:
Should you check that permissions was stored or just that no exception during permissions storing?